methods.ts

```diff
@@ -1447,10 +1462,8 @@ export type ApiMethods<F> = {
     is_personal?: boolean;
     /** Pass the offset that a client should send in the next query with the same text to receive more results. Pass an empty string if there are no more results or if you don't support pagination. Offset length can't exceed 64 bytes. */
     next_offset?: string;
-    /** If passed, clients will display a button with specified text that switches the user to a private chat with the bot and sends the bot a start message with the parameter switch_pm_parameter */
-    switch_pm_text?: string;
-    /** Deep-linking parameter for the /start message sent to the bot when user presses the switch button. 1-64 characters, only A-Z, a-z, 0-9, _ and - are allowed. */
-    switch_pm_parameter?: string;
```

Comment:
I suppose if anyone was using these parameters, it would be considered (type-only) breaking for them.

Reply:
Correct. It will be announced the same way as last time.